### PR TITLE
Fix preset

### LINKED ISSUE #129: Presets ID
The preset ID should be set in the presetcontroller, and the createPresetHandler should use this preset ID for saving the preset image. The camera ID in a preset should be set correctly in the createPresetHandler.

### PR DESCRIPTION
**Pull request by bvanwijk**
This fixes #129 and #131

### CODE REVIEW

File: backend/src/main/java/com/benine/backend/database/MySQLDatabase.java [hunk 1]
```diff
@@ -287,7 +282,7 @@ public String createAddSqlQuery(Preset preset) {
     if (preset.isAutoiris()) {
       autoir = 1;
     }
-    return "INSERT INTO presetsdatabase.presets VALUES(" + presetID + ","
+    return "INSERT INTO presetsdatabase.presets VALUES(" + preset.getCameraId() + ","
```
Comment (egedebruin) [inline backend/src/main/java/com/benine/backend/database/MySQLDatabase.java]: Shouldn't this be preset.getId()?

Comment (bvanwijk) [inline backend/src/main/java/com/benine/backend/database/MySQLDatabase.java]: Nice find, you are right I will change it

File: backend/src/main/java/com/benine/backend/http/CreatePresetHandler.java [hunk 2]
```diff
@@ -54,6 +54,8 @@ public void handle(String s, Request request, HttpServletRequest req, HttpServle
       getLogger().log(e.getMessage(), LogEvent.Type.WARNING);
       respondFailure(request, res);
     } catch (SQLException e) {
+      e.printStackTrace();
```
Comment (NaomidR) [inline backend/src/main/java/com/benine/backend/http/CreatePresetHandler.java]: Do you need e.printStackTrace() here?

**Comment by egedebruin** [comment]
Thanks for fixing, it looks good now. I agree with merging.

**Comment by jochem725** [comment]
Seems like a good fix. I agree with merging too.

**Comment by NaomidR** [comment]
I agree with merging this now

**Pull request merged**